cleanlab_studio/studio/studio.py

```diff
@@ -141,19 +144,19 @@ def apply_corrections(self, cleanset_id: str, dataset: Any, keep_excluded: bool
             cl_cols = self.download_cleanlab_columns(
                 cleanset_id, to_spark=False, include_project_details=True
             )
-            corrected_ds: snowpark.DataFrame = apply_corrections_snowpark_df(
+            snowflake_corrected_ds: snowpark.DataFrame = apply_corrections_snowpark_df(
                 dataset, cl_cols, id_col, label_col, keep_excluded
             )
-            return corrected_ds
+            return snowflake_corrected_ds
```

Comment:
Could this be from studio team? 
in that case we need to merge their main branch first 
@aditya1503 will have a look

Reply:
yes need to rebase everything here against latest master branch